discovery-provider/src/api/v1/playlists.py

```diff
@@ -195,13 +207,18 @@ def get(self):
     choices=("album", "playlist"),
     description="The collection type",
 )
+top_parser.add_argument(
+    "mood",
+    required=False,
+    description="Filer to a mood",
+)
 
 
-@ns.route("/top", doc=False)
+@full_ns.route("/top", doc=False)
```

Comment:
is it weird that playlists endpoint can return both albums and playlists?

Reply:
what do you mean? There's no separate /album endpoints
For this, the `type` is specified to be album or playlist

Reply:
top lives in v1/playlists, that's all. this is probably fine. maybe we should think about calling this stuff collection 🤷